tangelo/algorithms/classical/mp2_solver.py

```diff
@@ -85,3 +97,46 @@ def get_rdm(self):
         two_rdm = self.mp2_fragment.make_rdm2()
 
         return one_rdm, two_rdm
+
+    def get_mp2_params(self):
+        """Computes the MP2 initial variational parameters. Compute the initial
```

Comment:
I think we need to be careful about our definition and the function name. 

Are we "computing initial parameters for a variational algorithm" here (which sounds like an application, not a definition?) or are we computing single and double amplitudes associated to something that has some chemical meaning, output following a particular convention.... that can be used as initial parameters for variational approaches such as <...> ? 

Ideally you'd want to start with something that explains what we are actually computing before saying what they can be used for.